features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/components/MessagesReactionButton.kt

```diff
@@ -138,27 +137,24 @@ private fun TextContent(
 @Composable
 private fun IconContent(
     @DrawableRes resourceId: Int,
-    modifier: Modifier = Modifier
 ) = Icon(
     resourceId = resourceId,
     contentDescription = stringResource(id = R.string.screen_room_timeline_add_reaction),
     tint = ElementTheme.materialColors.secondary,
-    modifier = modifier
+    modifier = Modifier
         .size(ADD_EMOJI_SIZE)
 )
 
 @Composable
 private fun ReactionContent(
     reaction: AggregatedReaction,
-    modifier: Modifier = Modifier,
 ) = Row(
     verticalAlignment = Alignment.CenterVertically,
-    modifier = modifier,
 ) {
     // Check if this is a custom reaction (MSC4027)
     if (reaction.key.startsWith("mxc://")) {
         AsyncImage(
-            modifier = modifier
+            modifier = Modifier
```

Comment:
Note that `modifier` was used twice here, which is not correct.

Reply:
It's weird the linters never found the issue.

Reply:
Yes, maybe because of the form `= Row(`. I may check later...